_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,297 @@
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20861][] BIP 350: Implement Bech32m and use it for v1+ segwit addresses FIXME:Xekyo
+
+- [Bitcoin Core #21141][] updates the `-walletnotify` configuration
+  setting that calls a user-specified command each time a transaction is
+  seen that affects a loaded wallet.  Two new placeholders are added to
+  the arguments that can be passed to the command, `%b` for the hash of
+  a block containing the transaction and `%h` for the height of the
+  block.  Both are set to placeholder values for unconfirmed
+  transactions.
+
+- [C-Lightning #4428][] deprecates the `fundchannel_complete` RPC's
+  acceptance of txids, requesting instead that a [PSBT][topic psbt] be
+  passed.  The PSBT can be checked to ensure it contains the funding
+  output, eliminating a [problem][c-lightning #4416] where a user who
+  passes incorrect data can lose the ability to recover their funds.
+
+- [C-Lightning #4421][] Experimental option option_shutdown_wrong_funding: help me, I screwed up! FIXME:dongcarl (suggestion: link to last week's newsletter about this topic)
+
+- [LND #5064][] allows setting the desired number of blocks it should
+  take for a channel mutual close transaction to confirm.  This value is
+  used by the fee estimator to choose an appropriate priority fee for
+  the close transaction.
+
+- [LND #5068][] makes available a number of new configuration options
+  for limiting how much network gossip information LND processes.  This
+  can help on systems with limited resources.
+
+- [Libsecp256k1 #831][] implements an algorithm that can speed up
+  signature verification by 15%. It can also reduce by 25% the amount of
+  time it takes to generate signatures while still using a constant-time
+  algorithm that maximizes sidechannel resistance.  It also reduces
+  Libsecp256k1's dependencies on other libraries.  See [Newsletter
+  #136][news136 safegcd] for more information about this optimization.
+
+- [BIPs #1059][] adds [BIP370][] specifying version 2 PSBTs as
+  previously discussed on the mailing list (see [Newsletter
+  #128][news128 psbtv2]).
```

Comment:
I wonder if "version 0 PSBT" in that link should be updated to "version 1 PSBT"

Reply:
They deliberately decided to skip version 1.  BIP174 says "[...] the specification for version 0 of this format. "  BIP370 says "PSBT version 2".  :man_shrugging: